src/_gettsim/transfers/rente.py

```diff
@@ -359,34 +355,24 @@ def age_of_retirement(
         See basic input variable :ref:`jahr_renteneintr <jahr_renteneintr>`.
     monat_renteneintr
         See basic input variable :ref:`monat_renteneintr <monat_renteneintr>`.
-    rentner
-        See basic input variable :ref:`rentner <rentner>`.
-
 
     Returns
     -------
     Age at retirement.
 
     """
-    if rentner:
```

Comment:
Because `jahr_renteneintr` and `monat_renteneintr` are required to build the DAG independent of the value of `rentner` the distinction doesn't make much sense here.

Reply:
Then we are missing a test. Seems to me that this is the only way to get a `np.nan` in there.

Reply:
Okay, if zero is consistent, that is fine with me. But it should be documented and fixed using a test.